src/schema/city/index.ts

```diff
@@ -29,6 +29,18 @@ import { StaticPathLoader } from "lib/loaders/api/loader_interface"
 import { BodyAndHeaders } from "lib/loaders"
 import { sponsoredContentForCity } from "lib/sponsoredContent"
 
+const PartnerShowPartnerType = new GraphQLEnumType({
+  name: "PartnerShowPartnerType",
+  values: {
+    GALLERY: {
+      value: ["Gallery"],
+    },
+    MUSEUM: {
+      value: ["Institution", "Institutional Seller"],
```

Comment:
Museum has a [specific meaning](https://artsy.slack.com/archives/CFNFWD5F0/p1551904279181400?thread_ts=1551903577.176500&cid=CFNFWD5F0) for this use case.